src/batch/src/executor/row_seq_scan.rs

```diff
@@ -308,19 +308,15 @@ impl<S: StateStore> RowSeqScanExecutor<S> {
             .into_iter()
             .partition(|x| x.pk_prefix.size() == table.pk_indices().len());
 
+        let mut data_chunk_builder = DataChunkBuilder::new(table.schema().data_types(), chunk_size);
         // Point Get
-        let point_gets = select_all(point_gets.into_iter().map(|point_get| {
+        for point_get in point_gets {
```

Comment:
Is it changed from concurrent execution to sequential execution? Reason?

Reply:
Yes, the reason is fully concurrent multi point get can overwhelm our storage and they cause high contention if the number of point get is large like 1000 or even more.

Reply:
We use `UpstreamHashShard` because they are so similar and both of them are consistent hash. As an distribution, we can use it to enforce some plan nodes to require them providing the distribution we want.